apps/ledger-live-desktop/src/renderer/components/CustomImage/ImageGrayscalePreview.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import { Flex } from "@ledgerhq/react-ui";
+import { createCanvas } from "./imageUtils";
+import { ImageBase64Data, ImageDimensions } from "./types";
+import { targetDimensions } from "./shared";
+import ContrastChoice from "./ContrastChoice";
+import { ImageProcessingError } from "@ledgerhq/live-common/customImage/errors";
```

Comment:
i don't know why my linter is not complaining, but this import should be before the local imports 🤷